mala/common/parameters.py

```diff
@@ -687,6 +693,17 @@ class ParametersRunning(ParametersBase):
     inference_data_grid : list
         List holding the grid to be used for inference in the form of
         [x,y,z].
+
+    use_mixed_precision : bool
+        If True, mixed precision computation (via AMP) will be used.
+
+    training_report_frequency : int
+        Determines how often detailed performance info is printed during
+        training (only has an effect if the verbosity is high enough).
+
+    profiler_range : list
+        List with two entries determining with which batch size the CUDA
```

Comment:
I think this should be "iteration range" instead of "batch size" in the documentation here. Might also want to mention that this option is only relevant when running using the `nsys` profiler with the `--capture-range cudaProfilerApi` option enabled.

Reply:
Oh yes, you are right, I worded that weird.